open-craft/opencraft
# Apply SASS styles from agency

#### Testing

1. `make rundev`
2. Go to `/registration/`, check that everything works and looks right
3. Try to log in, make sure everything works and looks right
4. Go to the instance manager, make sure everything works and looks right
